client/src/catalog-error.html

```diff
@@ -6,21 +6,61 @@
   <template>
     <style include="catalog-styles"></style>
 
-    <h3>Error [[error.status]]</h3>
+    <h3>Error [[error.status]] - [[_code]]</h3>
     <h1 class="page-title">[[error.title]]</h1>
     <h2 class="page-subtitle">[[error.message]]</h2>
 
     <template is="dom-if" if="[[error.githubPath]]">
       <div>Would you like to view <a href="https://github.com/[[error.githubPath]]">[[error.githubPath]]</a> on Github instead?</div>
     </template>
+
+    <template is="dom-if" if="[[_codedError(_code, 5)]]">
+      <h2 class="page-subtitle">Invalid license</h2>
+
+      The GitHub repository must be licensed under an <a href="https://opensource.org/licenses/alphabetical">Open Source Inititative (OSI) approved license</a>. Please include a valid license or alternatively include an OSI approved license as an SPDX identifier (eg. "Apache-2.0") in the license field in the bower.json.
```

Comment:
Leaving this as is because we actually do 2 checks which is a bit inconsistent:
 * First on the [default/master branch](https://github.com/webcomponents/webcomponents.org/blob/master/src/manage.py#L256)
 * On every [version ingestion](https://github.com/webcomponents/webcomponents.org/blob/master/src/manage.py#L651)